### PR TITLE
simplify conda setup, use current best practices; fixes #421

### LINKED ISSUE #421: simplify installation with conda/pip
Drop conda support! just use pip!

Or should we?

### PR DESCRIPTION
**Pull request by sbenthall**
This simplifies the conda installation situation and brings it up to best practices:

- uses an `environment.yml` with _some_ but not all pinned dependencies
- `conda-setup.sh` script now installs pip requirements from `requirements.txt` list directly, with appropriate update strategy for conda compatibility

This reduces the number of places where we need to update pip dependencies.

### CODE REVIEW

**Comment by nllz** [comment]
Hmmmm does this mean we completely drop setting up (and working in) a virtual environment? Asking this because creating an environment is now not a part of the installation anymore. Will this get us in trouble with software package versions down the road?

**Comment by nllz** [comment]
Not building for me, see this gist for error:

https://gist.github.com/nllz/28aad34154f109794fe06236b314ee24

Implies I should run this command as sudo, not sure we want that? Or else we need to put it into the instructions. @sbenthall

**Comment by sbenthall** [comment]
We still mention virtual environment setup in the `pip` instructions. For conda, the conda environment replaces the pip environment.

I'm not sure how that error is happening.
Have you tried googling about it?

**Comment by nllz** [comment]
Yeah - the solution is running to 'sudo' so will add that to the instructions. Will first experiment with the notebooks to see all is working.

**Comment by sbenthall** [comment]
I'm a little concerned about this approach because a lot of people would find it very concerning to run a bash script, as opposed to an individual command, with sudo.

I could be wrong, but I believe this gets to the question of why one might want to use conda install instead of pip.

For packages with c binaries, I think sudo is necessary for a pip install.

I think it might not be for a conda install.

**Comment by nllz** [comment]
Seb - you were right! It was because of environment setting on my machine - on a Ubuntu liveCD with a clean install there were no issues whatsoever. So I removed sudo but made all pythons now python3 so everything runs cleanly.

File: README.md [hunk 1]
```diff
@@ -106,6 +94,17 @@ If you are interested in participating in BigBang development or would like supp
...
 
+## Troubleshooting 
+
+If the installation described above does not work, you can try to run the installation with Pip:
+
+```bash
+git clone https://github.com/datactive/bigbang.git
+# optionally create a new virtualenv here
+pip3 install -r requirements.txt
+python setup.py develop
```
Comment (sbenthall) [inline README.md:105]: python3

**Pull request merged**